components/automate-ui/src/app/pages/service-groups/service-groups.component.html

```diff
@@ -4,8 +4,8 @@
   <chef-subheading>Group of Habitat services running the same package and configuration.</chef-subheading>
 </chef-page-header>
 <div class="page-body">
-  <chef-status-filter-group>
-    <chef-option class="filter general" value="general" (click)="statusFilter('total')">
+  <chef-status-filter-group [value]="selectedStatus$ | async">
```

Comment:
This line is using the `selectedStatus` Observable to make it persistent on reload. Before we were not using it at all.